tasks/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: marketplacesvm-push-disk-images
   labels:
-    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/version: "0.1.1"
```

Comment:
It's not that important, but I think it should be 0.2.0 - it's a new feature, not a bug fix.

Reply:
Changed